tool/ui.re

```diff
@@ -230,13 +230,13 @@ impl Ui {
             color: COLOR_BLACK,
             x: 0.,
             y: 0.,
-            scale: SETTINGS.ui_scale,
+            scale: SETTINGS.ui_scale / 2.,
             scale_position: true,
         });
         let mut i = 0;
         for element in self.elements {
             let color = if self.selected == i { COLOR_RED } else { COLOR_BLACK };
-            element.draw(12. + i.to_float() * 12., color);
+            element.draw(48. + i.to_float() * 48., color);
```

Comment:
Using magical numbers directly can be hard to understand. Could you please refactor them out into a constant (well, currently variable in rebo) and name it to better indicate the intent?